Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.controller.ts

```diff
@@ -307,6 +328,52 @@
                     }
                 }
 
+                function bindCriticalities() {
+                    
+                    const optionMap = criticalityOptions.reduce((acc, next, _) => {
+                        acc[next.Id] = {
```

Comment:
I would bet we could refactor this block (lin 334-344) into a helper since it works on the generic optionType `Kitos.Models.IOptionEntity`

Then we can save 10 lines for each time we need to reduce a list of option types into a lookup dictionary containing those select 2 options